modules/selenium/src/test/java/org/testcontainers/junit/CustomWaitTimeoutWebDriverContainerTest.java

```diff
@@ -0,0 +1,27 @@
+package org.testcontainers.junit;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testcontainers.containers.BrowserWebDriverContainer;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+/**
+ *
+ */
+public class CustomWaitTimeoutWebDriverContainerTest extends BaseWebDriverContainerTest {
+
+    @Rule
+    public BrowserWebDriverContainer chromeWithCustomTimeout = new BrowserWebDriverContainer<>()
+            .withDesiredCapabilities(DesiredCapabilities.chrome())
+            .withStartupTimeout(Duration.of(30, SECONDS));
+
+    @Test
+    public void simpleTest() throws IOException {
+        doSimpleWebdriverTest(chromeWithCustomTimeout);
```

Comment:
No assertion in this test unfortunately; I couldn't think of a way to do it that wasn't over-engineered. In manual testing I've dialled this right down to 1 second to verify that the timeout is having an effect.